iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -212,6 +216,8 @@ public class HiveIcebergStorageHandler implements HiveStoragePredicateHandler, H
   public static final String MERGE_ON_READ = "merge-on-read";
   public static final String STATS = "/stats/snap-";
 
+  public static final String TABLE_DEFAULT_LOCATION = "TABLE_DEFAULT_LOCATION";
```

Comment:
What is the TABLE_DEFAULT_LOCATION here? It would not be a default path? Would not be /user/hive/warehouse?

Reply:
``TABLE_DEFAULT_LOCATION`` is the constant that we will replace the default location with, say /user/hive/warehouse/database/table1 is the default location, so in the metadata.json path, this will be replaced by ``TABLE_DEFAULT_LOCATION``

so it will look like TABLE_DEFAULT_LOCATION/metadata/abc-metadata.json

That is what Janos use case is: In ranger he can configure policies to allow  paths within ``TABLE_DEFAULT_LOCATION``